client/pages/form-pages/authorize.tsx

```diff
@@ -0,0 +1,111 @@
+import { Form, Formik, FormikProps } from 'formik';
+import { Field, FormStepTitles, FormSteps, Button } from '@components';
+import { AuthorizationInterface } from 'constants/interfaces';
+import SignaturePad from 'react-signature-canvas';
+import SignatureCanvas from 'react-signature-canvas';
+
+const initialValues = {
+  projectManagerSignature: '',
+  financialOfficerSignature: '',
+};
+
+const SignPad: React.FC<any> = () => {
+  const [dataURL, setDataURL] = useState<string | null>(null);
+  const [showSign, setShowSign] = useState<boolean | null>(false);
+  let padRef = useRef<SignatureCanvas>(null);
+  const clear = () => {
+    padRef.current?.clear();
+  };
+  const trim = () => {
+    const url = padRef.current?.getTrimmedCanvas().toDataURL('image/png');
+    if (url) setDataURL(url);
+    setShowSign(false);
+  };
+  return (
+    <div className='w-full'>
+      {dataURL ? (
+        <div className='w-full flex justify-between p-4'>
+          <img className={'sigImage'} src={dataURL} alt='user generated signature' />
+        </div>
+      ) : null}
+      {showSign && (
+        <SignaturePad
+          ref={padRef}
+          canvasProps={{ className: 'sigCanvas w-full border-2 mb-2 border-gray-200' }}
+        />
+      )}
+      <div className='sigPreview flex justify-between'>
+        {!showSign ? (
+          <>
+            <div></div>
+            <Button variant='outline' onClick={() => setShowSign(true)}>
+              Sign
+            </Button>
+          </>
+        ) : (
+          <>
+            <Button variant='outline' onClick={clear}>
+              Clear
+            </Button>
+            <Button variant='primary' onClick={trim}>
+              Save
+            </Button>
+          </>
+        )}
+      </div>
+    </div>
+  );
+};
+
+export const AuthorizationPage: React.FC = () => {
+  return (
+    <Formik initialValues={initialValues} onSubmit={() => {}}>
+      {({ values }: FormikProps<AuthorizationInterface>) => (
+        <Form className='flex justify-center'>
+          <div className='w-2/4 p-4 gap-y-6 bg-white flex flex-col items-center'>
+            <div className='mb-4 flex items-center flex-col'>
+              <h1 className='text-xl font-medium text-bcBluePrimary'>
+                {FormStepTitles[FormSteps.AUTHORIZATION]}
+              </h1>
+              <h3>Please proivde detailed information for your application.</h3>
+            </div>
+
+            <p className='font-bold'>
+              We certify that the information contained in this application, is to the best of our
+              knowledge correct, complete and has been submitted with the concurrence of our Council
+              or Board.
+            </p>
+
+            <Field
+              name='projectManagerSignature'
+              type='text'
+              label='Project Manager, Name and Signature'
+              maxLength={300}
+            ></Field>
+            <SignPad />
```

Comment:
Have we looking into the accessibility implications of this field? Is it a required field? I can't imagine a keyboard or SR user would be able to fill this out very easily

Reply:
No we have looked into it. Was not part of original requirement but would be good to have.